app/assets/javascripts/channels/notifications.js

```diff
@@ -0,0 +1,18 @@
+App.notifications = App.cable.subscriptions.create("NotificationsChannel", {
+  received(data) {
+    let html = `<div class='flash flash_alert'>
+                  <p> ${I18n.t('admin.members.js.end_of_auto_enrollment')} </p>
```

Comment:
If this is the only place that you use I18n in JS, you could maybe try to rename this file into `notifications.js.erb` and see if the asset pipeline compiles it properly.
If it does, then you could do
```
                  <p> <%= I18n.t('admin.members.js.end_of_auto_enrollment') %> </p>
```
and get rid of the `i18n-js` gem entirely